Source/Particles/PhysicalParticleContainer.H

```diff
@@ -185,7 +195,6 @@ public:
         const amrex::Real q_tot, long npart, const int do_symmetrize);
 
     /** Load a particle beam from an external file
-     *
      * @param[in] q_tot total charge of the particle species to be initialized
```

Comment:
It's not directly related to this change, but can you add the documentation for the z_shift input parameter?

Reply:
It's very good that you spotted this!
I thought I had added it in the other PR.